nixos/modules/hardware/ledger.nix

```diff
@@ -0,0 +1,46 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.hardware.ledger;
+
+  udevRules = pkgs.stdenv.mkDerivation {
```

Comment:
I think this should be packaged normally, plus hydra wouldn't be able to cache this, even if it's just copying a file.

Reply:
Ahh realized this was done in the [previous pr](https://github.com/NixOS/nixpkgs/pull/39950/files#diff-e1d434c7cc9829ac6eedb7781b49a403).

Reply:
Well this PR supersedes the other one, so your “criticism” still stands.